modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -57,3 +57,31 @@ The Operator is installed automatically into the `openshift-operators` namespace
 ====
 +
 . Verify that the *Status* is set to *Succeeded Up to date*  to confirm successful installation of {pipelines-title} Operator.
++
+[WARNING]
+====
+The success status may show as *Succeeded Up to date* even if installation of other components is in-progress. Therefore, it is important to verify the installation manually in the terminal.
+====
++
+. Verify that all components of the {pipelines-title} Operator were installed successfully. Login to the cluster on the terminal, and run the following commands:
++
+----
+$ oc get tektonconfig config
+NAME     VERSION   READY   REASON
+config   1.9.2     True 
+
+$ oc get tektonpipeline,tektontrigger,tektonaddon,pac
+NAME                                          VERSION   READY   REASON
+tektonpipeline.operator.tekton.dev/pipeline   v0.41.1   True 
+NAME                                        VERSION   READY   REASON
+tektontrigger.operator.tekton.dev/trigger   v0.22.2   True    
+NAME                                    VERSION   READY   REASON
+tektonaddon.operator.tekton.dev/addon   1.9.2     True 
+NAME                                                             VERSION   READY   REASON
+openshiftpipelinesascode.operator.tekton.dev/pipelines-as-code   v0.15.5   True 
+----
```

Comment:
Please revise the formatting here per the OCP docs guidelines:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Each command and output should be a separate code block